### PR TITLE
Support for Bankscrap v2.0.0

### PR DESCRIPTION
**Pull request by javiercr**
- Refactored initialize
- Renamed `dni` to `nif`
- Renamed bundle_login, as now the `initialize` method in the base `Bank` class is already calling `login`.
- Build account and transaction using money objects

@raulmarcosl I did the required changes to support the new `Bank#initialize` from Bankscrap `v2.0.0`, however I can't test them as I don't have an ING account. Can you test it and merge it if everything is ok?

Make sure to also test fetching investments. Thanks!

Also, this adapter is always returning accounts and transactions with `EUR` as currency. Can we get that data from the API?

### CODE REVIEW

**Comment by javiercr** [comment]
@raulmarcosl 👀

File: lib/bankscrap/ing/bank.rb [hunk 1]
```diff
@@ -18,19 +18,12 @@ class Bank < ::Bankscrap::Bank
       SAMPLE_HEIGHT    = 30
       SAMPLE_ROOT_PATH = '/numbers'.freeze
 
-      def initialize(user, password, log: false, debug: false, extra_args:)
-        @dni      = user
-        @password = password.to_s
-        @birthday = extra_args.with_indifferent_access['birthday']
-        @log      = log
-        @debug    = debug
+      REQUIRED_CREDENTIALS  = [:nif, :password, :birthday]
```
Comment (raulmarcosl) [inline lib/bankscrap/ing/bank.rb]: I don't like the `:nif` credential. [Following this](https://es.wikipedia.org/wiki/N%C3%BAmero_de_identificaci%C3%B3n_fiscal), the proper credential should be `:dni`.

**Comment by raulmarcosl** [comment]
I've checked this with my account and it works smoothly, great job!

**Comment by javiercr** [comment]
@raulmarcosl just renamed `:nif` to `:dni`, could you test it again, merge the PR and push the gem if it works? Thanks! 🎉

**Pull request merged**